appveyor.yml

```diff
@@ -59,6 +59,10 @@ install:
 - if %TESTING%==Func verifier /standard /driver winfsp-x64.sys & exit 0
 - ps: Restart-Computer -Force
 - ps: Start-Sleep -s 60
+- ps: |
```

Comment:
This should happen before we schedule the reboot. See line `ps: Restart-Computer -Force` above.